modules/home-age.nix

```diff
@@ -0,0 +1,233 @@
+{ config
```

Comment:
I kind of think we should name this module age-home.nix so that it sorts near age.nix. Not that strong of an opinion though, so if there is some reason or precedent to have it this way, okay.

Reply:
The only reason I picked this name is that it was the one used in a previous PR.

Another possibility is to name the NixOS module `nixos.nix`/`nixos/default.nix`, and the HM module `home-manager.nix`/`home-manager/default.nix` (this is what sops-nix does).

Will rename it to `age-home.nix`.